src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,15 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid(private val color: Color,
+                         private val height: Double,
+                         private val top: Double,
+                         private val bottom: Double) : Figure(color) {
+    override fun draw() {
+        println("Figure: isosceles trapezoid, area: ${calculateArea()} sq. units, " +
```

Comment:
You may use triple quote marks here to avoid concatenation

Reply:
Please check link below, in case if you have additional questions here
https://kotlinlang.org/docs/strings.html#multiline-strings